test/metabase/models/channel_test.clj

```diff
@@ -0,0 +1,22 @@
+(ns metabase.models.channel-test
+  (:require
+   [clojure.test :refer :all]
+   [metabase.api.channel-test :as api.channel-test]
+   [metabase.test :as mt]
+   [metabase.util.encryption :as encryption]
+   [metabase.util.encryption-test :as encryption-test]
+   [toucan2.core :as t2]))
+
+(comment
+ ;; to register the :metabase-test channel implementation
+ api.channel-test/keepme)
+
+(deftest channel-details-is-encrypted
+  (encryption-test/with-secret-key "secret"
+   (mt/with-model-cleanup [:model/Channel]
+     (let [channel (t2/insert-returning-instance! :model/Channel {:name    "Test channel"
+                                                                  :type    "channel/metabase-test"
```

Comment:
Lots of extra lime breaks

Reply:
🍋? :D 

I don't see any line breaks tho.

Reply:
Blame my phone screen 😄